### PR TITLE
create a testimonial section

### PR DESCRIPTION
**Pull request by DrKimpatrick**
#### What does this PR do?
 - Creates a testimonial section

#### Description of Task to be completed?
 - Provide sample testimonials from sample students

#### Screenshots (if appropriate)
testimonial

### CODE REVIEW

File: src/components/testimonials/card.scss [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    h2 {
+      color: #141313;
+      font-weight: 300;
+      font-size: 14px;
+      line-height: 16px;
+      margin: auto;
+    }
+
+    h3 {
```
Comment (younggeeks) [inline src/components/testimonials/card.scss]: Can we extract these headers to classes we can reuse?

**Review by younggeeks — CHANGES_REQUESTED** [review]

File: src/components/testimonials/card.scss (hunk 1 above)
Comment (younggeeks) [reply src/components/testimonials/card.scss]: we could have primary, secondary header classes for reusability

File: src/components/testimonials/card.scss [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      font-weight: 500;
+      font-size: 17px;
+      line-height: 20px;
+      color: #706262;
+      margin: 0 auto;
+    }
+  }
+}
+
+.one {
```
Comment (younggeeks) [inline src/components/testimonials/card.scss]: We could do better in descriptive class names

**Review by younggeeks — CHANGES_REQUESTED** [review]

File: src/components/testimonials/card.scss (hunk 1 above)
Comment (DrKimpatrick) [reply src/components/testimonials/card.scss]: @younggeeks This has been resolved

File: src/components/testimonials/card.scss (hunk 2 above)
Comment (DrKimpatrick) [reply src/components/testimonials/card.scss]: @younggeeks This has been resolved

**Pull request merged**